### ISSUE TITLE
Basic version of kyma istio operator reacting on CR creation

### ISSUE THREAD

**Issue by strekm** [kyma-project/istio#4]
**Description**

Implement first version of kyma istio operator which reacts on creation of CR. Istioctl should NOT be used to install Istio. It is possible to configure `numTrustedProxies`. Applying profiles is out of scope of this task. Base Istio configuration from kyma should used for installation.

AC:
- [x] kyma istio operator installs istio based on conf in Istio CR
- [x] documentation created
- [x] manual test case created

**Reasons**

Modularisation of kyma components

**Attachments**
https://github.com/kyma-project/istio/blob/main/charts/istio-operator/crds/apiextensions.k8s.io_v1_customresourcedefinition_istios.operator.kyma-project.io.yaml
https://github.com/kyma-project/kyma/blob/main/resources/istio/templates/istio-operator.yaml
depends on: https://github.com/orgs/kyma-project/projects/27/views/1

**Comment by strekm** [comment]
for sake of simplicity we assume that IstioOperatorFile CR is hardcoded in Istio operator. Version of Istio is also hardcoded.
